Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -618,17 +623,15 @@ extension KMConversationViewController {
     }
 
     private func showRatingView() {
-        KMCustomEventHandler.shared.publish(triggeredEvent: CustomEvent.rateConversationClick, data: nil)
         guard self.ratingVC == nil else { return }
         let ratingVC = RatingViewController()
         ratingVC.closeButtontapped = { [weak self] in
             self?.hideRatingView()
         }
         ratingVC.feedbackSubmitted = { [weak self] feedback in
             print("feedback submitted with rating: \(feedback.rating)")
-            KMCustomEventHandler.shared.publish(triggeredEvent: CustomEvent.submitRatingClick, data: ["UserSelection": ["SubmittedFeedback": feedback]])
+            KMCustomEventHandler.shared.publish(triggeredEvent: CustomEvent.submitRatingClick, data:  ["rating": feedback.rating.rawValue,"comment":feedback.comment ?? "","conversationId": self?.viewModel.channelKey])
```

Comment:
See if we need to make it backward compatible or add a note in migration (major release) that it will be removed in coming release.

Reply:
Adding migration guide in release notes as well as our doc section